packages/cli/src/spec/inputHelpersSpec.ts

```diff
@@ -18,169 +18,175 @@ import * as inputHelpers from '../lib/inputHelpers';
       const validate = inputHelpers.createValidateString(3);
-      expect(validate('123').isOk()).toBeTrue();
-      expect(validate('1234').isOk()).toBeTrue();
+      expect((await validate('123')).isOk()).toBeTrue();
+      expect((await validate('1234')).isOk()).toBeTrue();
     });
 
-    it('Fails validations above maximum length', () => {
+    it('Fails validations above maximum length', async () => {
       const validate = inputHelpers.createValidateString(undefined, 3);
-      expect(validate('1234').isError()).toBeTrue();
-      expect(validate('12345').isError()).toBeTrue();
+      expect((await validate('1234')).isError()).toBeTrue();
+      expect((await validate('12345')).isError()).toBeTrue();
     });
 
-    it('Passes validations below or equal to maximum length', () => {
+    it('Passes validations below or equal to maximum length', async () => {
       const validate = inputHelpers.createValidateString(undefined, 3);
-      expect(validate('123').isOk()).toBeTrue();
-      expect(validate('12').isOk()).toBeTrue();
-      expect(validate('1').isOk()).toBeTrue();
-      expect(validate('').isOk()).toBeTrue();
+      expect((await validate('123')).isOk()).toBeTrue();
+      expect((await validate('12')).isOk()).toBeTrue();
+      expect((await validate('1')).isOk()).toBeTrue();
+      expect((await validate('')).isOk()).toBeTrue();
     });
 
-    it('minLength and maxLength can be applied at the same time', () => {
+    it('minLength and maxLength can be applied at the same time', async () => {
       const validate = inputHelpers.createValidateString(2, 3);
-      expect(validate('').isError()).toBeTrue();
-      expect(validate('1').isError()).toBeTrue();
-      expect(validate('12').isOk()).toBeTrue();
-      expect(validate('123').isOk()).toBeTrue();
-      expect(validate('1234').isError()).toBeTrue();
+      expect((await validate('')).isError()).toBeTrue();
+      expect((await validate('1')).isError()).toBeTrue();
+      expect((await validate('12')).isOk()).toBeTrue();
+      expect((await validate('123')).isOk()).toBeTrue();
+      expect((await validate('1234')).isError()).toBeTrue();
     });
   });
 
   describe('#validatePackageId', () => {
-    it('returns Ok for valid package ids', () => {
-      expect(inputHelpers.validatePackageId('com.pwa_directory.appspot.com').isOk()).toBeTrue();
-      expect(inputHelpers.validatePackageId('com.pwa1directory.appspot.com').isOk()).toBeTrue();
-    });
-
-    it('returns Error for invalid package ids', () => {
-      expect(inputHelpers.validatePackageId('com.pwa@directory.appspot.com').isError()).toBeTrue();
-      expect(inputHelpers.validatePackageId('com..example').isError()).toBeTrue();
-      expect(inputHelpers.validatePackageId('').isError()).toBeTrue();
-      expect(inputHelpers.validatePackageId('com.1char.twa').isError()).toBeTrue();
+    it('returns Ok for valid package ids', async () => {
```

Comment:
nit: indentation on the lines bellow

Reply:
That's what I did at the beginning but lint didn't like it for some reason

Reply:
Try 4 spaces instead of 2. Also, make sure the line is not hitting the 100 char limit.